spec/factories/tasks.rb

```diff
@@ -5,5 +5,10 @@
     from = Date.parse("2019/08/01")
     to   = Date.parse("2019/12/31")
     deadline { Random.rand(from..to) }
+
+    trait :due_yesterday do
+      status { 'done' }
```

Comment:
String型のカラムではないので、enumの値で定義しておきましょう。
```suggestion
      status { :done }
```

Reply:
修正いたしました。
commit IDは、 e2cd20f 　になります。